substrate/client/transaction-pool/tests/zombienet/network-specs/asset-hub-high-pool-limit-fatp.toml

```diff
@@ -0,0 +1,64 @@
+[relaychain]
+# single-state
+# fork-aware
+default_image = "parity/polkadot:latest"
+default_command = "polkadot"
+chain = "rococo-local"
+
+[[relaychain.nodes]]
+name = "alice"
+rpc_port = 9944
+validator = true
+
+[[relaychain.nodes]]
+name = "bob"
+validator = true
+
+[[parachains]]
+id = 2000
+chain = "asset-hub-rococo-local"
+default_command = "polkadot-parachain"
+default_image = "parity/polkadot-parachain:latest"
+cumulus_based = true
+default_args = [
+	"--force-authoring",
+	"--pool-kbytes 2048000",
+	"--pool-limit 500000",
+	"--pool-type=fork-aware",
+	"--rpc-max-connections 15000",
+	"--rpc-max-response-size 150",
+	"--rpc-max-subscriptions-per-connection=128000",
+	"--state-pruning=1024",
+	"-laura::cumulus=debug",
+	"-lbasic-authorship=debug",
+	"-lpeerset=trace",
+	"-lsub-libp2p=debug",
+	"-lsync=trace",
+	"-ltxpool=trace",
+]
+devAccounts = [
```

Comment:
@pepoviola 
why not use `parachain.genesis.runtimeGenesis.patch...` here?
Would `devAccounts` be compatible with `zombienet` binary?

Reply:
oh, it seems it is used in the other file: `substrate/client/transaction-pool/tests/zombienet/network-specs/asset-hub-low-pool-limit-fatp.toml`

This one looks like leftover.